app/src/main/java/org/sopt/pingle/presentation/ui/joingroup/JoinViewModel.kt

```diff
@@ -54,15 +72,39 @@ class JoinViewModel : ViewModel() {
 
     private fun getIsSelected(position: Int) = _joinGroupSearchData.value[position].isSelected.get()
 
-    init {
-        _joinGroupData.value = JoinGroupCodeEntity(
-            id = 0,
-            keyword = "연합동아리",
-            name = "SOPT",
-            meetingCount = 10,
-            participantCount = 200
-        )
+    fun getJoinGroupInfo(teamId: Int) {
+        _joinGroupCodeState.value = UiState.Loading
+        viewModelScope.launch {
+            _joinGroupCodeState.value = UiState.Loading
+            runCatching {
+                getJoinGroupInfoUseCase.invoke(teamId = teamId).collect { joinGroupInfo ->
```

Comment:
```suggestion
                getJoinGroupInfoUseCase(teamId = teamId).collect { joinGroupInfo ->
```
유즈케이스에서 invoke를 재정의 했기 때문에 이렇게 사용 가능합니다 나머지도 고쳐주세오